### DISCUSSION TITLE
Autotune Mode for inav

### DISCUSSION THREAD

**Discussion by Seeelefant** [iNavFlight/inav#5888] [Ideas]
## Current Behavior

Currently PID etc. tuning is a huge manual effort.

## Desired Behavior
Implement AUTOTUNE mode like in Arducopter: https://ardupilot.org/copter/docs/autotune.html

## Suggested Solution

"AutoTune attempts to automatically tune the Stabilize P, Rate P and D, and maximum rotational accelerations to provide the highest response without significant overshoot. Copter needs to be “basically” flyable in AltHold mode before attempting to use AutoTune as the feature needs to be able to “twitch” the copter in the roll and pitch axis."

[a comment by an automated account is not shown]

**Comment by Black6spdZ** [comment]
didn't it used to be in INAV?

**Comment by Seeelefant** [comment]
As far as I know only for planes.
